sspl_test/common.py

```diff
@@ -103,7 +106,8 @@ def init_messaging_msg_processors():
             logger.info("SSPL-Tests Starting %s" % curr_module.name())
             curr_module._set_debug(True)
             thread = Thread(target=_run_thread_capture_errors,
-                            args=(curr_module, msgQlist, conf_reader, product))
+                            args=(curr_module, msgQlist, conf_reader, product,
+                                  alerts_on_csm))
```

Comment:
alerts_on_csm is only needed by ingressprocessor, no need to pass to all processor.
If there are complexities into passing to just one file, then I am fine with this